loic/style.css

```diff
@@ -32,4 +32,12 @@ Tags: blog, education, news, one-column, two-columns, left-sidebar, wide-blocks,
 a {
 	text-decoration-thickness: .0625em !important;
 	text-underline-offset: .15em;
+}
+
+/*
+ * SyntaxHighlighter styles
+ * https: //github.com/Automattic/themes/issues/8387
```

Comment:
Nit: I noticed a space between `https:` and `//`. You might want to correct it for someone to copy and paste.